evil.sh

```diff
@@ -56,6 +56,11 @@ annoying && alias cat=true;
 # Use a random sort option whenever `ls` is invoked.
 annoying && function ls { command ls -$(opts="frStu"; echo ${opts:$((RANDOM % ${#opts})):1}) "$@"; }
 
+Taunt OS X users with random words through their speakers
+annoying && if [ "$(uname)" = 'Darwin' ]; then
+  sh -c 'sleep $[($RANDOM % 900) + 300]s; say $(head -$(($RANDOM % $(wc -l < /usr/share/dict/words) + 1)) /usr/share/dict/words | tail -2);' &
```

Comment:
You could instead of reading the whole file twice and doing ugly shell things use `shuf(1)` depending on whether that's available on Darwin.
An alternative would be a simple `sort -R /usr/share/dict/words | head -1` (although preferable `head -n 1`).